packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -54,6 +62,9 @@ class PackagesCommand extends FlutterCommand {
 
   @override
   Future<FlutterCommandResult> runCommand() async => FlutterCommandResult.fail();
+
+  @override
+  void printUsage() => _usagePrintFn(usage);
```

Comment:
This is inspired by the custom devices command, which also bypasses arg's direct use of `print` by [overriding `printUsage`](https://github.com/flutter/flutter/blob/835b892d7fbe808be92d5eeee7b31c7bb7a3b1eb/packages/flutter_tools/lib/src/commands/custom_devices.dart#L161).